packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1476,8 +1528,117 @@ class RenderPhysicalModel extends _RenderCustomClip<RRect> {
     super.debugFillProperties(description);
     description.add(new DiagnosticsProperty<BoxShape>('shape', shape));
     description.add(new DiagnosticsProperty<BorderRadius>('borderRadius', borderRadius));
-    description.add(new DoubleProperty('elevation', elevation));
-    description.add(new DiagnosticsProperty<Color>('color', color));
+  }
+}
+
+/// Creates a physical shape layer that clips its child to a [ShapeBorder].
```

Comment:
it doesn't actually, does it? it clips to a Path.

Reply:
done